crest/Assets/Crest/Crest/Shaders/OceanInputs/Resources/CopyDepthIntoCache.shader

```diff
@@ -59,7 +59,7 @@ Shader "Crest/Copy Depth Buffer Into Cache"
 
 			float4 frag(Varyings input) : SV_Target
 			{
-				float deviceDepth = SAMPLE_DEPTH_TEXTURE(_CamDepthBuffer, input.uv);
+				float deviceDepth = tex2D(_CamDepthBuffer, input.uv);
```

Comment:
Should this have a swizzle for the red channel to avoid truncation warnings on some platforms?

Reply:
good point done